### ISSUE TITLE
Add Export/Import other programs' splits

### ISSUE THREAD

**Issue by okiyama** [Zingler/Msplit#1]
Exporting the splits that you got on MSplit to either WSplit or Llanfair format would be good. Saving these to SD card as well as standard sharing options (email, bluetooth et cetera) would be ideal.

Importing from other splits would also be nice.

**Comment by Zingler** [comment]
Yeah, I though about this as well. Down the road, I would like to add this when I find a good solution to the sharing aspect. Some people might not be satisfied with dragging files to and from the sd card, and email might be clunky. 

Integrating with other programs always creates dependencies and maintenance on the app as well.

**Comment by okiyama** [comment]
That's a good point. It shouldn't be a problem for WSplit because that won't be seeing an update in the foreseeable future, but Llanfair would be problematic. I've talked to that dev and he is planning on big updates in the near future that might break backwards compatibility with splits, who knows.

Is there any way that I can contact you directly? I'd like to work on this but I want to know what features to be going at.
